statsmodels/discrete/discrete_model.py

```diff
@@ -2074,10 +2074,8 @@ def _score_nbin(self, params, Q=0):
         a1 = 1/alpha * mu**Q
         # Note: Q is either 0 or 1, so a1 is either 1/alpha or mu/alpha.
         if Q: # nb1
-            digamma_part = (
-                        special.digamma(y+mu/alpha)
-                        - special.digamma(mu/alpha)
-                        )
+            digamma_part = (special.digamma(y + mu / alpha)
+                        - special.digamma(mu / alpha))
```

Comment:
align below parenthesis, unless we need more space if parenthesis is already far to the right
(next time)